Disher/View Controllers/RecipesViewController.m

```diff
@@ -46,35 +40,38 @@ - (void)viewDidLoad {
 //}
 
 - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
-    return self.mealDBresults.count;
+    return self.spoonResults.count + self.mealDBresults.count;
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
     RecipeCell *cell = [tableView dequeueReusableCellWithIdentifier:@"RecipeCell" forIndexPath:indexPath];
     
-    // Configure the cell...
-    cell.recipeName.text = self.mealDBresults[indexPath.row][@"strMeal"];
-    cell.recipeDescription.text = self.mealDBresults[indexPath.row][@"strArea"];
-    NSString *imageLink = self.mealDBresults[indexPath.row][@"strMealThumb"];
-    NSURL *imageURL = [NSURL URLWithString:imageLink];
-    
-    [cell.recipeImage setImageWithURL:imageURL];
+    // loop through mealdb results first, then go to spoonacular results
+    if (indexPath.row < self.mealDBresults.count) {
+        cell.recipeName.text = self.mealDBresults[indexPath.row][@"strMeal"];
+        NSString *imageLink = self.mealDBresults[indexPath.row][@"strMealThumb"];
+        NSURL *imageURL = [NSURL URLWithString:imageLink];
+        [cell.recipeImage setImageWithURL:imageURL];
+    }
+    else if (indexPath.row >= self.mealDBresults.count) {
```

Comment:
this condition is exactly the opposite as above, so a simple if/else (without condition) would suffice, i believe